src/main/kotlin/uk/gov/justice/digital/hmpps/personrecord/model/Person.kt

```diff
@@ -1,30 +1,21 @@
 package uk.gov.justice.digital.hmpps.personrecord.model
 
-import com.fasterxml.jackson.annotation.JsonIgnoreProperties
-import com.fasterxml.jackson.annotation.JsonInclude
-import io.swagger.v3.oas.annotations.media.Schema
 import uk.gov.justice.digital.hmpps.personrecord.client.model.offender.OffenderDetail
 import uk.gov.justice.digital.hmpps.personrecord.jpa.entity.PersonIdentifierEntity
 import uk.gov.justice.digital.hmpps.personrecord.model.hmcts.commonplatform.Defendant
 import uk.gov.justice.digital.hmpps.personrecord.model.hmcts.event.LibraHearingEvent
 import uk.gov.justice.digital.hmpps.personrecord.model.identifiers.CROIdentifier
 import uk.gov.justice.digital.hmpps.personrecord.model.identifiers.PNCIdentifier
+import uk.gov.justice.digital.hmpps.personrecord.model.types.ContactType
 import uk.gov.justice.digital.hmpps.personrecord.model.types.SourceSystemType
 import java.time.LocalDate
-import java.util.UUID
+import java.util.*
 
-@JsonInclude(JsonInclude.Include.NON_NULL)
-@JsonIgnoreProperties(ignoreUnknown = true)
 data class Person(
-  @Schema(description = "The unique person identifier", example = "f4165b62-d9eb-11ed-afa1-0242ac120002")
   val personId: UUID? = null,
-  @Schema(description = "A person's first name", example = "Guinevere")
   val givenName: String? = null,
-  @Schema(description = "A person's middle name(s)", example = "Catherine Anne")
   val middleNames: List<String>? = emptyList(),
-  @Schema(description = "A person's surname", example = "Atherton")
   val familyName: String? = null,
-  @Schema(description = "A person's date of birth", example = "1972-08-27")
   val dateOfBirth: LocalDate? = null,
```

Comment:
👍

Reply:
Should also be able to remove `implementation("io.swagger.core.v3:swagger-annotations:2.2.21")` from `build.gradle.kts`